### PR TITLE
Gha more changes

### PR DESCRIPTION
**Pull request by pondrejk**

### CODE REVIEW

**Comment by pondrejk** [comment]
@mshriver @JacobCallahan sadly, the problem still persists

File: scripts/customer_scenarios.py [hunk 1]
```diff
@@ -40,67 +41,65 @@ def get_bz_data(paths):
...
+        'id',
+        'summary',
+        'status',
+        'resolution',
+        'cf_last_closed',
+        'flags',
+        'external_bugs',
     ]
-    bz_url = "https://bugzilla.redhat.com"
+    bz_url = 'https://bugzilla.redhat.com'
```
Comment (Gauravtalreja1) [inline scripts/customer_scenarios.py:52]:
```suggestion
    bz_url = settings.bugzilla.url
```

**Comment by JacobCallahan** [comment]
@pondrejk can you add a check in your script to validate whether or not the bz key is being populated in dynaconf? Don't print it out, just check to see if it isn't None and has a len > 0

**Comment by ColeHiggins2** [comment]
Did we get this to work after adding the checks?

**Comment by pondrejk** [comment]
Hello, I'm working on a rewrite, the current implementation won't work as github doesn't allow to access secrets from forks

**Comment by JacobCallahan** [comment]
> Hello, I'm working on a rewrite, the current implementation won't work as github doesn't allow to access secrets from forks 

Lets discuss this more during the JFT call

**Comment by pondrejk** [comment]
Some update on progress, I changed the workflow in various ways:
- it now checks bzs for all tests, not just for files changed in the latest PR
- as such it is not tied to merge or pr action but it is scheduled to run weekly
- if it finds missing tags, it opens an issue with a list of tests to address (assigned to me for the time being)

This all has been tested in my fork. The reason I'm keeping this in the draft state is that I'm getting some false positives when querying bugzilla for external bugs, I'm investigating the problem, and will share updates here

**Comment by pondrejk** [comment]
The issue mentioned above is now fixed, pr is ready for review. @JacobCallahan @mshriver @Gauravtalreja1 @ColeHiggins2

**Review by JacobCallahan — APPROVED** [review]

File: scripts/customer_scenarios.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    bz_fields = [
+        'id',
+        'summary',
+        'status',
+        'resolution',
+        'cf_last_closed',
+        'external_bugs',
+    ]
+    bz_url = settings.bugzilla.url
+    bz_key = settings.bugzilla.api_key
```
Comment (jyejare) [inline scripts/customer_scenarios.py:56]: Need not to initialize the vars as you have used it only once, you can direct use it :)

Non Blocking comment!

**Review by jyejare — APPROVED** [review]

**Review by latran — APPROVED** [review]
ack

**Pull request merged**